### PR TITLE
feat: [modal migration] submit article

### PR DESCRIPTION
**Pull request by el**
@rebelchris  @sshanzel Can you please check this locally? I was not able to test all scenarios.

### CODE REVIEW

File: packages/shared/src/components/modals/SubmitArticleModal.tsx [hunk 1]
```diff
@@ -182,66 +181,78 @@ export default function SubmitArticleModal({
...
-    <ResponsiveModal
-      {...modalProps}
-      onRequestClose={onRequestClose}
-      padding={false}
-      style={{ content: { maxWidth: '35rem', width: '100%' } }}
-    >
-      <header className="flex justify-between items-center px-6 w-full h-14 border-b border-theme-divider-tertiary">
-        <div className="flex flex-row items-center">
-          <h3 className="font-bold typo-title3">{headerCopy}</h3>
+    <form ref={submitFormRef} aria-busy={isValidating} onSubmit={onSubmit}>
```
Comment (rebelchris) [inline packages/shared/src/components/modals/SubmitArticleModal.tsx:184]: I'm not quite sure I like the form wrapping around the whole modal.
Can we not move it into the body and use form="id" on the button rather?

File: packages/shared/src/components/modals/SubmitArticleModal.tsx [hunk 2]
```diff
@@ -182,66 +181,78 @@ export default function SubmitArticleModal({
...
+      <Modal
+        {...modalProps}
+        kind={Modal.Kind.FlexibleCenter}
+        size={Modal.Size.Medium}
+        onRequestClose={onRequestClose}
+      >
+        <Modal.Header>
+          <Modal.Header.Title>{headerCopy}</Modal.Header.Title>
           <BetaBadge />
```
Comment (rebelchris) [inline packages/shared/src/components/modals/SubmitArticleModal.tsx:193]: We can remove this, after discussion on Slack:
https://dailydotdev.slack.com/archives/C02AA18RD1D/p1668588454326879

**Review by rebelchris — CHANGES_REQUESTED** [review]
I'm not fully able to test UX wise, but tested all variations locally.

The only issue seems to be with the existing article see attached:
Screenshot 2022-11-16 at 12 00 52

File: packages/shared/src/components/modals/SubmitArticleModal.tsx (hunk 1 above)
Comment (el) [reply packages/shared/src/components/modals/SubmitArticleModal.tsx:184]: Sure thing, but what is the downside?

File: packages/shared/src/components/modals/SubmitArticleModal.tsx (hunk 2 above)
Comment (el) [reply packages/shared/src/components/modals/SubmitArticleModal.tsx:193]: Done 👍

File: packages/shared/src/components/modals/SubmitArticleModal.tsx (hunk 1 above)
Comment (el) [reply packages/shared/src/components/modals/SubmitArticleModal.tsx]: Done 👍

Comment (rebelchris) [reply packages/shared/src/components/modals/SubmitArticleModal.tsx]: No technical downside more of a structural point of view.
It felt like abusing the wrapping.

**Comment by el** [comment]
@rebelchris @sshanzel here is how it looks now:

image

**Comment by el** [comment]
Updated article position:
image

**Review by rebelchris — CHANGES_REQUESTED** [review]
Would you mind checking with a article with a super short name?
It seems that proves a issue. (might have been existing)

**Comment by el** [comment]
Short title fix

image

**Review by rebelchris — APPROVED** [review]
Amazing, nice work 🥳

**Pull request merged**